libCacheSim/dataStructure/hashtable/chainedHashtable.c

```diff
@@ -196,13 +196,13 @@ cache_obj_t *chained_hashtable_insert(hashtable_t *hashtable, request_t *req) {
     cache_obj->hash_next = new_cache_obj;
     cache_obj = new_cache_obj;
   }
-  hashtable->n_obj += 1;
+  __sync_fetch_and_add(&hashtable->n_obj, 1);
```

Comment:
neat, `__sync_fetch_and_add` is legacy code according to GCC doc (https://gcc.gnu.org/onlinedocs/gcc/_005f_005fsync-Builtins.html), it might be better to use `__atomic_add_fetch`

Reply:
Yes, __sync_fetch_and_add has been outdated. However, when I used <stdatomic.h> and atomic_fetch_add, I got an error: ‘_Atomic’ does not name a type. I'm not sure whether it is caused by my GCC version. Maybe you can help me to solve this problem :)